pyccel/ast/core.py

```diff
@@ -2439,6 +2442,49 @@ def _sympystr(self, printer):
         value = sstr(self.value)
         return '{0}={1}'.format(name, value)
 
+class TupleVariable(Variable):
+
+    """Represents a valued variable in the code.
```

Comment:
Since you are giving a proper docstring to this new class, I think it is worth following [Numpy's dosctring guidelines](https://numpydoc.readthedocs.io/en/latest/format.html#sections), so that one day we will be able to create automatic documentation for Pyccel. In particular you should add the header
```
Parameters
----------
```
and add a dashed line below "Examples".